web/tests/functional/cppcheck/test_proj/lib.h

```diff
@@ -0,0 +1,12 @@
+// -----------------------------------------------------------------------------
+//                     The CodeChecker Infrastructure
+//   This file is distributed under the University of Illinois Open Source
+//   License. See LICENSE.TXT for details.
+// -----------------------------------------------------------------------------
+
+// core.DivideZero (C, C++, ObjC)
+// Check for division by zero.
+
+void div(int x) {
+  int y = x % 0; // warn
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'y' is assigned a value that is never used.](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4623161)